dts/arm/st/n6/stm32n6.dtsi

```diff
@@ -362,6 +362,39 @@
 			};
 		};
 
+		fdcan1: can@5000a000 {
+			compatible = "st,stm32h7-fdcan";
```

Comment:
Perhaps it's time to come up with a new `compatible` for this integration of the Bosch M_CAN? If any of you have any insight in what to name this, I am all ears. The "h7" name was chosen since it was the first family (Zephyr-wise) where this integration was supported.

Reply:
@henrikbrixandersen In Bosch M_CAN user manual on page 5: https://www.bosch-semiconductors.com/media/ip_modules/pdf_2/m_can/mcan_users_manual_v331.pdf register map is defined. Since a device follows this map "bosch,m-can" would be a good choise as you suggest.